index.bs

```diff
@@ -2432,17 +2432,15 @@ Note: Extensions should aim to define authenticator arguments that are as small
 ## <dfn>Client extension processing</dfn> ## {#sctn-client-extension-processing}
 
 Extensions may define additional processing requirements on the client platform during the creation of credentials or the
-generation of an assertion. In order for the [=[RP]=] to verify the processing took place, or if the processing has a result
-value that the [RP] needs to be aware of, the extension specifies a extension key-value pair to be included in the [=client data=] extensions field,
-which is returned to the [RP] via {{AuthenticatorResponse/clientDataJSON}}.
-
-If any extension processed by a client defines such a
-value, the client includes a dictionary in its [=client data=] with the key {{CollectedClientData/extensions}}. For each such
+generation of an assertion.
+The [=client extension input=] for the extension is used an input to this client processing.
+The [=client extension input=] value may be any value that can be encoded using JSON.
```

Comment:
I'd rather see the restriction that extensions must define their client extension input as a JSON'able value, by the definition of [client extension input](https://api.csswg.org/bikeshed/?url=https://raw.githubusercontent.com/selfissued/webauthn/mbj-extensions-connective-tissue/index.bs#client-extension-input), rather than in the client processing section.